fintech-examples/fintech-impl/pom.xml

```diff
@@ -29,37 +29,42 @@
         <dependency>
             <groupId>io.swagger.core.v3</groupId>
             <artifactId>swagger-annotations</artifactId>
-            <version>2.1.1</version>
+            <version>${io.swagger.annotations.version}</version>
         </dependency>
         <dependency>
             <groupId>javax.ws.rs</groupId>
             <artifactId>javax.ws.rs-api</artifactId>
-            <version>2.1.1</version>
+            <version>${javax.ws.rs-api.version}</version>
         </dependency>
         <dependency>
             <groupId>org.glassfish.jersey.core</groupId>
             <artifactId>jersey-client</artifactId>
-            <version>2.30</version>
+            <version>${glassfish.jersey-client.version}</version>
         </dependency>
         <dependency>
             <groupId>org.glassfish.jersey.media</groupId>
             <artifactId>jersey-media-json-jackson</artifactId>
-            <version>2.30</version>
+            <version>${glassfish.jersey-client.version}</version>
         </dependency>
         <dependency>
             <groupId>org.glassfish.jersey.media</groupId>
             <artifactId>jersey-media-multipart</artifactId>
-            <version>2.30</version>
+            <version>${glassfish.jersey-client.version}</version>
```

Comment:
Why jersey is used in swagger-codegen
`<library>jersey2</library>`
instead of spring?
(`<library>spring-mvc</library>`)

Reply:
spring-mvc is for server stub code.
with <language> "java" rather than "spring" 
<library>spring-mvc does not exist:

[WARNING] Output directory does not exist, or is inaccessible. No file (.swagger-codegen-ignore) will be evaluated.
[ERROR] 
java.lang.RuntimeException: Unknown library: spring-mvc
Available libraries:

Reply:
I need client stub code, that why I use jersey.